modules/oe_theme_content_entity_organisation/oe_theme_content_entity_organisation.install

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Module install file.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+
+/**
+ * Implements hook_install().
+ */
+function oe_theme_content_entity_organisation_install() {
```

Comment:
Such code exists in 

- oe_theme_content_entity_venue_install()
- oe_theme_content_entity_contact_install()
- oe_theme_content_event_install()
- oe_theme_content_project_install()

Might be we should use one function from oe_theme (oe_theme_set_configuration(array $configurations)) to set configuration instead of duplicating code?

Reply:
yes, thats a good idea, include the isConfigSyncing check, with parameters or FileStorage path and displays

Reply:
We should do this indeed, feel free to propose a refactoring ticket about it.

Reply:
Ticket has been created - https://citnet.tech.ec.europa.eu/CITnet/jira/browse/D8TEWPPE-53.